client/api/volume/v1beta3/api.proto

```diff
@@ -10,27 +10,52 @@ service Volume {
     rpc ListVolumesOnDisk(ListVolumesOnDiskRequest) returns (ListVolumesOnDiskResponse) {}
     // MountVolume mounts the volume at the requested global staging path
     rpc MountVolume(MountVolumeRequest) returns (MountVolumeResponse) {}
-    // DismountVolume gracefully dismounts a volume
-    rpc DismountVolume(DismountVolumeRequest) returns (DismountVolumeResponse) {}
-    // IsVolumeFormatted checks if a volume is formatted with NTFS
+    // UnmountVolume gracefully unmounts a volume
+    rpc UnmountVolume(UnmountVolumeRequest) returns (UnmountVolumeResponse) {}
+    // IsVolumeFormatted checks if a volume is formatted.
     rpc IsVolumeFormatted(IsVolumeFormattedRequest) returns (IsVolumeFormattedResponse) {}
-    // FormatVolume formats a volume with the provided file system
+    // FormatVolume formats a volume with NTFS.
     rpc FormatVolume(FormatVolumeRequest) returns (FormatVolumeResponse) {}
     // ResizeVolume performs resizing of the partition and file system for a block based volume
     rpc ResizeVolume(ResizeVolumeRequest) returns (ResizeVolumeResponse) {}
+    // GetVolumeStats gathers DiskSize, VolumeSize and VolumeUsedSize for a volume
```

Comment:
update comments "gathers total types and used bytes for a volume?"

Reply:
fixed